static_src/test/functional/user_invite.spec.js

```diff
@@ -53,4 +58,22 @@ describe('User roles', function () {
       expect(userInviteElement.getValidatorMessage()).toMatch(bottomError);
     });
   });
+
+  describe('A user on page for an space', function () {
+    it('should navigates to org X space XX', function () {
+      userRoleElement = new UserRoleElement(browser, browser.element('.test-users'));
+      browser.url(urlOrgXSpaceXX);
+      browser.waitForExist('.test-users');
+    });
+
+    it('should not have the user invite panel', function () {
+      userRoleElement.setAndGetUserRole(cookieManagerOrgXSpaceXX);
+      browser.url(urlOrgXSpaceXX);
+
+      browser.waitForExist('.test-users');
+      var ells = $$('.test-users-invite');
```

Comment:
we can use the `waitForExist` with the reverse flag. http://webdriver.io/api/utility/waitForExist.html